lib/reporting/identity_verification_report.rb

```diff
@@ -81,10 +83,10 @@ def to_csv
         csv << ['Workflow completed - In-Person Pending', idv_final_resolution_in_person]
         csv << ['Workflow completed - Fraud Review Pending', idv_final_resolution_fraud_review]
         csv << []
-        csv << ['Succesfully verified', successfully_verified_users]
-        csv << ['Succesfully verified - Inline', idv_final_resolution_verified]
-        csv << ['Succesfully verified - GPO Code Entry', gpo_verification_submitted]
-        csv << ['Succesfully verified - In Person', usps_enrollment_status_updated]
+        csv << ['Successfully verified', successfully_verified_users]
+        csv << ['Successfully verified - Inline', idv_final_resolution_verified]
+        csv << ['Successfully verified - GPO Code Entry', gpo_verification_submitted]
+        csv << ['Successfully verified - In Person', usps_enrollment_status_updated]
```

Comment:
I happened to notice this and it bugged me, but I want to make sure no one is comparing old to new CSVs in an automated way where this would become a breaking change.

Reply:
let's break these typo fixes into a separate PR since they're not strictly needed for this new report, which can pull the values off programatically

Reply:
I have reverted these changes, so they can come in a later PR